src/types/internal.d.ts

```diff
@@ -0,0 +1,27 @@
+export type Nothing = null;
+export type Maybe<T> = T | Nothing;
+
+export type Map<K extends string | number, V> = {
+  [k in K]: V
+}
+
+export type Dictionary<T> = {
```

Comment:
Now that we have a generic types folder, do you mind importing it here:

https://github.com/TwinePlatform/twine-api/blob/cbe6de1efff3c3a2bc2237bf9b19b32fbd0d66aa/src/auth/permissions/permissions.ts#L3

Reply:
Done